library/ssl_tls13_keys.c

```diff
@@ -677,9 +682,9 @@ int mbedtls_ssl_tls13_calculate_verify_data( mbedtls_ssl_context* ssl,
     MBEDTLS_SSL_DEBUG_BUF( 4, "handshake hash", transcript, transcript_len );
 
     if( from == MBEDTLS_SSL_IS_CLIENT )
-        base_key = ssl->handshake->tls13_hs_secrets.client_handshake_traffic_secret;
+        base_key = tls13_hs_secrets->client_handshake_traffic_secret;
```

Comment:
One last bit, please move this at line 667 such a `base_key` is always set to a value different from NULL when going to `exit` label.

Reply:
base_key is inited as `sever_handshake_traffic_secret`